blackbox/docs/installation.txt

```diff
@@ -1,105 +1,107 @@
+.. highlight:: sh
+
 ============
 Installation
 ============
 
-This section shows you how to install CrateDB on a single node.
+This page documents the most basic sort of CrateDB installation.
+
+.. NOTE::
+
+   CrateDB comes pre-packaged for many Linux systems, Docker, and cloud hosting
+   providers. There is also a handy one-line command that let's you download
+   and run CrateDB on Mac OS X.
 
-To set up a multi-node cluster, you will need to tell CrateDB how to find the
-other nodes. See the `Multi Node Setup`_ article for more information.
+   To learn more, check out the `install section`_ of the CrateDB Guide.
 
-Java (JVM) Version
-==================
+Prerequisites
+=============
 
 CrateDB requires `Java 8`_. We recommend using `Oracle's Java`_ on OSX and
 OpenJDK_ on Linux Systems.
+
 The same JVM version must be used on all CrateDB nodes.
 
 We recommend installing `Java 8`_ update 20 or later.
 
 .. _install_targz:
 
-Install Via Tarball
-===================
+Download
+========
+
+Download `the latest CrateDB release`_.
+
+.. TIP::
+
+   Older releases of CrateDB are `also available`_.
 
-.. highlight:: sh
 
-After downloading the latest stable version of CrateDB from
-`crate.io/download/ <https://crate.io/download/>`_
-the package needs to be unpacked::
+Once downloaded, unpack the tarball and change into the resulting directory::
 
-   sh$ tar zxf crate-*.tar.gz
+   sh$ tar -xzf crate-*.tar.gz
    sh$ cd crate-*
 
-Once unpacked CrateDB can be started in the foreground like this::
+Run
+===
+
+There is no need to configure or build anything.
+
+Once unpacked, CrateDB can be started in the foreground like this::
 
    sh$ ./bin/crate
```

Comment:
I think it would make sense here to say that this runs a single instance of CrateDB that is bound the local IP.